dds_cli/utils.py

```diff
@@ -53,13 +53,19 @@ def calculate_magnitude(projects, keys, iec_standard=False):
             # exclude values smaller than base, such that empty projects don't interfer with
             # the calculation ensures that a minimum can be calculated if no val is larger than base
             minimum = (lambda x: min(x) if x else 1)([val for val in values if val >= base])
-            mag = 0
+            maximum = (lambda x: max(x) if x else 1)([val for val in values if val >= base])
```

Comment:
```suggestion
minimum, maximum = (lambda x: min(x), max(x) if x else (1,1))([val for val in values if val >= base])
```

Reply:
Thank you very much for the suggestion. Indeed much cleaner! I was eager to prove my point that it is a quick fix in the code that I choose the code duplication approach over carefully thinking it through...

Reply:
I changed it accordingly. I couldn't directly commit your suggestion though, because suggestions on deleted lines are apparently not supported. 

But thanks a lot: Very elegant and much cleaner!